packages/components/src/hooks/index.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Created a `hooks` subfolder that all hooks for components would be kept in - similar to what Playbooks does